check-log/lib/check-log.go

```diff
@@ -444,6 +451,8 @@ func getStateFile(stateDir, f string, args []string) string {
 	)
 }
 
+var errValidStateFileNotFound = fmt.Errorf("state file not found, or corrupted")
+
 func getBytesToSkip(f string) (int64, error) {
 	state, err := loadState(f)
```

Comment:
If state file is corrupted, `loadState()` will return `nil, nil`.
Is it intended behavior that `getBytesToSkipOld` will be called to try to read old state file in this case?

Reply:
I thought the behavior is not the best but acceptable.
But finally I decided not to read old state file the case ****